### PR TITLE
Added new create method

### PR DESCRIPTION
**Pull request by OakLoaf**

### CODE REVIEW

File: src/main/java/me/xemor/configurationdata/entity/EntityData.java [hunk 1]
```diff
@@ -136,6 +136,15 @@ public static EntityData create(ConfigurationSection configurationSection) {
         return create(configurationSection, EntityType.ZOMBIE);
     }
 
+    public static EntityData create(ConfigurationSection parentSection, String path, @NotNull EntityType def) {
+        if (parentSection.isString(path)) {
```
Comment (Xemorr) [inline src/main/java/me/xemor/configurationdata/entity/EntityData.java:145]: sorry about nitpicking again I feel bad with the reviews

Add a message if EntityType.valueOf fails. 

try {
  ...
} catch(IllegalArgumentException e) {
  blah blah logging blah blah you are a noob and entered an invalid entity at parentSection.getCurrentPath() + ".entity"
}

File: src/main/java/me/xemor/configurationdata/entity/EntityData.java [hunk 2]
```diff
@@ -86,7 +87,11 @@ private EntityData(EntityType entityType) {
     }
 
     public Entity spawnEntity(@NotNull Location location) {
-        Entity entity = location.getWorld().spawnEntity(location, entityType);
+        return spawnEntity(location.getWorld(), location);
+    }
```
Comment (Xemorr) [inline src/main/java/me/xemor/configurationdata/entity/EntityData.java:91]: Why did you re add this method?

File: src/main/java/me/xemor/configurationdata/entity/EntityData.java (hunk 1 above)
Comment (OakLoaf) [reply src/main/java/me/xemor/configurationdata/entity/EntityData.java:145]: You're totally fine, it's important to make sure it fits what you want for your plugins as it is your project :)

File: src/main/java/me/xemor/configurationdata/entity/EntityData.java (hunk 2 above)
Comment (OakLoaf) [reply src/main/java/me/xemor/configurationdata/entity/EntityData.java:91]: > Why did you re add this method?

It was being used throughout SkillsLibrary and it made me realise that it's probably safer to add this back. Equally happy to remove this again and just update in SkillsLibrary

Comment (Xemorr) [reply src/main/java/me/xemor/configurationdata/entity/EntityData.java:91]: I think I might have ported over myself (when I had to change the shading of adventure I had to update everything), do a pull on your skillslibrary

Comment (OakLoaf) [reply src/main/java/me/xemor/configurationdata/entity/EntityData.java:91]: Ahh, my branch is out of date

**Comment by Xemorr** [comment]
oh shit I merged before the world method could be (re)removed

**Pull request merged**